SU-HSDO/suhumsci
# SHS 5178 Ocean color scheme link contrast adjustments

# READY FOR REVIEW

## Summary
This PR adjusts a few different color contrast differences for the Ocean color scheme. 

## Urgency
medium

## Steps to Test
1. Set up local with archaeology and history sites, switched to HS Colorful theme, and using the Ocean color scheme. 
2. Observe the link contrast on More links with arrows and ensure they pass the color contrast tests:
https://archaeology.suhumsci.loc/academics/field-experience
3. Ensure that links and more links in "wells" (beige background areas) pass contrast tests:
https://history.suhumsci.loc/people/caroline-winterer
4. Ensure buttons with a filled background pass contrast tests (including on hover):
https://archaeology.suhumsci.loc/events/distinguished-lecture-series/gathering-shadows-material-things
5. Ensure buttons with a white background pass contrast tests (including on hover): 
See "SUNet Login" in the footer area on any page
6. Ensure "reset" button passes contrast tests (including on hover):
https://history.suhumsci.loc/events/upcoming-events
7. Ensure Pager link color contrast passes contrast tests:
https://history.suhumsci.loc/recent-news
8. Ensure that date cards have proper contrast:
https://history.suhumsci.loc/events/upcoming-events


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
